src/Freya.Core/Builder.fs

```diff
@@ -26,30 +26,30 @@ open Freya.Core.Operators
 
 (* Builder *)
 
-/// Defines a computation expression builder for constructing <see cref="Freya{T}" /> computations.
-type FreyaBuilder () =
+/// Defines a computation expression builder for constructing <see cref="Core{T}" /> computations.
+type CoreBuilder () =
```

Comment:
I would prefer `Builder` to `CoreBuilder`. I don't find `Core` sufficiently meaningful. Keep in mind that some people may only use `Freya.Core`, so names should still work and provide sufficient context for such users.